tock/hours/views.py

```diff
@@ -468,6 +470,7 @@ class AdminBulkTimecardSerializer(serializers.Serializer):
     )
 
 
+@user_passes_test(lambda u: u.is_authenticated, login_url='/')
```

Comment:
Nice!

Reply:
Is there any way for `login_url` to be specified in the Django settings, e.g. via [`settings.LOGIN_URL`](https://docs.djangoproject.com/en/2.0/ref/settings/#login-url)?  It'd help make things a bit more DRY.

Also, is there a reason the [`login_required`](https://docs.djangoproject.com/en/2.0/topics/auth/default/#the-login-required-decorator) decorator can't be used here?  It seems like it'd make the code a bit more readable but I dunno.